README.md

```diff
@@ -23,25 +23,41 @@ services in the exchange.
       - listen on all interfaces: `sed -i -e "s/#listen_addresses = 'localhost'/listen_addresses = '*'/" /usr/local/var/postgres/postgresql.conf`
       - `brew services start postgresql` or if it is already running `brew services restart postgresql`
     - Test: `psql "host=$MY_IP dbname=postgres user=<myuser> password=''"`
-- Add a config file on your development system at /etc/horizon/exchange/config.json with at least the following content (this is needed for the automated tests. Defaults and the full list of config variables are in `src/main/resources/config.json`):
+- Add a configuration file on your development system at `/etc/horizon/exchange/config.json` with at minimum the following content (this is needed for the automated tests. Defaults and the full list of configuration variables are in `src/main/resources/config.json`):
 
-```
-{
-	"api": {
-		"db": {
-			"jdbcUrl": "jdbc:postgresql://localhost/postgres",		// my local postgres db
-			"user": "myuser",
-			"password": ""
-		},
-		"logging": {
-			"level": "DEBUG"
-		},
-		"root": {
-			"password": "myrootpw"
-		}
-	}
-}
-```
+  ```
+  {
+    "api": {
+      "db": {
+        "jdbcUrl": "jdbc:postgresql://localhost/postgres",    // my local postgres db
+        "user": "myuser",
+        "password": ""
+      },
+      "logging": {
+        "level": "DEBUG"
+      },
+      "root": {
+        "password": "myrootpw"
+      }
+    }
+  }
+  ```
+- Add a plain-text `config/exchange-api.tmpl` configuration file template to your project's root directory. This template does not require any content if you do not wish to use it. Should you wish to use this template, minimal configuration is required (example below). The value for each key in the template corresponds to an environment variable passed to the container (i.e. Using a config-map in Kubernetes/Openshift). The Docker container will create a `/etc/horizon/exchange/config.json` configuration file based on the provided template and environment variables at container creation. To swap templates a new Docker image is required.
```

Comment:
Since this file gets built into the docker image, it should be committed in git (so 2 different developers building from the latest master get the same result). If you do that, you don't need this section telling them to add it.